.github/CONTRIBUTING.md

```diff
@@ -2,49 +2,47 @@
-* New Features
-  * If you'd like to accomplish something in the extension that it doesn't already do, describe the problem in a new Github Issue.
-    * Issues that have been identified as a feature request will be labeled `✨ enhancement`.
-    * If you'd like to implement the new feature, please wait for feedback from the project maintainers before spending
-      too much time writing the code. In some cases, `✨ enhancement`s may not align well with the project objectives at
-      the time.
-* Tests, Documentation, Miscellaneous
-  * If you think the test coverage could be improved, the documentation could be clearer, you've got an alternative
-    implementation of something that may have more advantages, or any other change we would still be glad hear about
-    it.
-    * If its a trivial change, go ahead and send a Pull Request with the changes you have in mind
-    * If not, open a Github Issue to discuss the idea first.
-* Snippets
-  * To add snippets:
-    * Add a directory in the `snippets` folder with the name of the language.
-    * Add one or more files in the language directory with snippets.
-    * Update the `package.json` to include the snippets you added.
+## Prerequisites
+
+Before submitting code to this project, you should first complete the following prerequisites. Completing these steps will make your first contribution easier:
+
+- Read the [Code of Conduct](https://github.com/deepgram/deepgram-go-sdk/blob/main/.github/CODE_OF_CONDUCT.md) at [https://github.com/deepgram/deepgram-go-sdk/blob/main/.github/CODE_OF_CONDUCT.md](https://github.com/deepgram/deepgram-go-sdk/blob/main/.github/CODE_OF_CONDUCT.md).
+- [Sign up](http://github.com/signup) for a GitHub user account.
+
+## Your First Contribution
 
-We also welcome anyone to work on any existing issues with the `👋🏽 good first issue` tag.
+The first step to getting starting contributing to Go SDK is to find something to work on. Help is always welcome, and no contribution is too small (but see below)!
 
-## Requirements
+Here are some things you can do today to get started contributing:
 
-For a contribution to be accepted:
+- Help improve the documentation
+- Clarify code, variables, or functions that can be commented on
+- Write test coverage
+- Help triage issues
 
-* The test suite must be complete and pass. Run `go test`
-* Code must follow existing styling conventions
-* Commit messages must be descriptive. Related issues should be mentioned by number.
+If the above suggestions don't appeal to you, you can browse the issues labeled as a [good first issue](https://github.com/deepgram/deepgram-go-sdk/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) to see who is looking for help. The good first issue label also indicates that project maintainers have committed to providing extra assistance for new contributors. When you've found an issue to work on, you can assign it to yourself.

```

Comment:
I THINK there is a way to do these links so they're absolute to the repo this is in, saving editing every time. I think it was Atom's README that taught me how to do it. Or VSCode. Can't remember. I'll have a look

Reply:
If there is, we can always revisit it, and I will update the template in the oss repo as well.